helmfile/charts/prometheus-alerts/templates/rules/fluentd.yaml

```diff
@@ -0,0 +1,86 @@
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.fluentd }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ printf "%s-%s" (include "prometheus-alerts.fullname" .) "fluentd" | trunc 63 | trimSuffix "-" }}
+  labels:
+    app: {{ template "prometheus-alerts.fullname" . }}
+{{ include "prometheus-alerts.labels" . | indent 4 }}
+{{- if .Values.defaultRules.labels }}
+{{ toYaml .Values.defaultRules.labels | indent 4 }}
+{{- end }}
+{{- if .Values.defaultRules.annotations }}
+  annotations:
+{{ toYaml .Values.defaultRules.annotations | indent 4 }}
+{{- end }}
+spec:
+  groups:
+  - name: fluentd
+    rules:
+    - alert: FluentdNodeDown
+      expr: up{job=~"fluentd.*"} == 0
+      for: 10m
+      labels:
+        service: fluentd
+        severity: warning
+      annotations:
+        summary: fluentd cannot be scraped
+        description: Prometheus could not scrape {{ "{{ $labels.job }}" }} for more than 10 minutes
+    - alert: FluentdNodeDown
```

Comment:
Is it smart to have these alerts the same name? Will they not be considered as one? Or maybe that's intentional?

As long as we're sure that the priority is updated when the next alert fires I'm fine with whatever

Reply:
According to the docs [any existing conflicting labels will be overwritten](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/).
From what I can see the kube-prometheus-stack default alerts are written in the same manner. ex: [NodeFilesystemAlmostOutOfFiles](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/upstream/kube-prometheus-stack/templates/prometheus/rules-1.14/node-exporter.yaml#L137)

If you want I can do a test for them.

Reply:
Cool! No I think that's fine. We can always rename them if we see that it doesn't work